core/modules/replica/DatabaseServices.h

```diff
@@ -123,7 +124,7 @@ class DatabaseServices
      *
      * The Performance object is explicitly passed as a parameter to avoid
      * making a call back to the request which may create a deadlock because
-     * method saveState() is meant to be called by the requests' implementations.
+     * method saveState() is meant to be called by the request's implementations.
```

Comment:
Does the request have multiple implementations? ("request's implementations")
or a single implementation ("request's implementation") (probably this is the one you want?)
ore are there different requests, with different implementations? "requests' implementations" (apostrophe after 'requests' is possessive of plural)

Reply:
Yes, this comment sounds a bit confusing. I would rather remove it altogether than trying to explain too much in this context.

Reply:
Done